website/src/pages/news.tsx

```diff
@@ -22,233 +22,65 @@ export default class NewsPage extends React.Component<any, any> {
             'Sourcegraph in the news.'
         return (
             <Layout location={this.props.location}
-                 meta={{
+                meta={{
                     title:
-                        'Sourcegraph news and press releases',
+                    'Sourcegraph news and press releases',
                     description:
-                        'Sourcegraph in the news.'
-                 }}
+                    'Sourcegraph in the news.'
+                }}
             >
-                <div className="news bg-white text-dark">
-                    <section>
-                        <div>
+            <div className="news bg-white text-dark">
+                <section>
+                    <div>
                         <ContentSection color="primary" className="hero-section text-center py-5">
```

Comment:
```suggestion
                        <ContentSection color="purple" className="hero-section text-center py-5">
```
How about we set the background in the news page to purple? It would be different from the events page and purple is a more alerting color than blue which is on par with the news theme.
![image](https://user-images.githubusercontent.com/9110008/76262774-746a0b80-621a-11ea-9ea1-c7ebc0b389e9.png)